README.md

```diff
@@ -87,6 +87,73 @@ docker run -p 9101:9101 quay.io/prometheus/haproxy-exporter:v0.9.0 --haproxy.scr
+| haproxy_exporter_build_info | A metric with a constant '1' value labeled by version, revision, branch, and goversion from which haproxy_exporter was built. |
+| haproxy_exporter_csv_parse_failures | Number of errors while parsing CSV. |
+| haproxy_exporter_total_scrapes | Current total HAProxy scrapes. |
+| haproxy_frontend_bytes_in_total | Current total of incoming bytes. |
+| haproxy_frontend_bytes_out_total | Current total of outgoing bytes. |
+| haproxy_frontend_current_session_rate | Current number of sessions per second over last elapsed second. |
+| haproxy_frontend_current_sessions | Current number of active sessions. |
+| haproxy_frontend_http_requests_total | Total HTTP requests. |
+| haproxy_frontend_http_responses_total | Total of HTTP responses. |
+| haproxy_frontend_limit_session_rate | Configured limit on new sessions per second. |
+| haproxy_frontend_limit_sessions | Configured session limit. |
+| haproxy_frontend_max_session_rate | Maximum observed number of sessions per second. |
+| haproxy_frontend_max_sessions | Maximum observed number of active sessions. |
+| haproxy_frontend_request_errors_total | Total of request errors. |
+| haproxy_frontend_requests_denied_total | Total of requests denied for security. |
+| haproxy_frontend_sessions_total | Total number of sessions. |
+| haproxy_server_bytes_in_total | Current total of incoming bytes. |
+| haproxy_server_bytes_out_total | Current total of outgoing bytes. |
+| haproxy_server_check_duration_milliseconds | Previously run health check duration, in milliseconds |
+| haproxy_server_check_failures_total | Total number of failed health checks. |
+| haproxy_server_connection_errors_total | Total of connection errors. |
+| haproxy_server_current_queue | Current number of queued requests assigned to this server. |
+| haproxy_server_current_session_rate | Current number of sessions per second over last elapsed second. |
+| haproxy_server_current_sessions | Current number of active sessions. |
+| haproxy_server_downtime_seconds_total | Total downtime in seconds. |
+| haproxy_server_http_responses_total | Total of HTTP responses. |
+| haproxy_server_max_queue | Maximum observed number of queued requests assigned to this server. |
+| haproxy_server_max_session_rate | Maximum observed number of sessions per second. |
+| haproxy_server_max_sessions | Maximum observed number of active sessions. |
+| haproxy_server_redispatch_warnings_total | Total of redispatch warnings. |
+| haproxy_server_response_errors_total | Total of response errors. |
+| haproxy_server_retry_warnings_total | Total of retry warnings. |
+| haproxy_server_sessions_total | Total number of sessions. |
+| haproxy_server_up | Current health status of the server (1 = UP, 0 = DOWN). |
+| haproxy_server_weight | Current weight of the server. |
+| haproxy_up | Was the last scrape of haproxy successful. |
+| http_request_duration_microseconds | The HTTP request latencies in microseconds. |
```

Comment:
These come from the exporter itself, they shouldn't be part of the list I think.